src/libsyntax/parse/lexer/mod.rs

```diff
@@ -1009,94 +969,6 @@ impl<'a> StringReader<'a> {
         let c = self.ch;
 
         match c.expect("next_token_inner called at EOF") {
-            'b' => {
-                self.bump();
-                let lit = match self.ch {
-                    Some('r') => self.scan_raw_byte_string(),
-                    _ => unreachable!(),  // Should have been a token::Ident above.
-                };
-                let suffix = self.scan_optional_raw_name();
-
-                Ok(token::Literal(lit, suffix))
-            }
-            'r' => {
-                let start_bpos = self.pos;
-                self.bump();
-                let mut hash_count: u16 = 0;
-                while self.ch_is('#') {
-                    if hash_count == 65535 {
-                        let bpos = self.next_pos;
-                        self.fatal_span_(start_bpos,
-                                         bpos,
-                                         "too many `#` symbols: raw strings may be \
-                                         delimited by up to 65535 `#` symbols").raise();
```

Comment:
seems like this error has "regressed" to relying on the plain integer conversions [here](https://github.com/rust-lang/rust/pull/59706/files/66161ec14ead25d373a9cfb2e285e5952d78a5f8..68d04bdf278b21c5b6c2765970eee2342d7975f6#diff-d06ad31c547d2d94c8b8770006977767R620)